src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,48 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val listOfGreetings: MutableList<String> = mutableListOf()
+
+        val englishImplementation = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world"
```

Comment:
FYI this function could be single lined, and it is better to not use hardcoded strings